dev/create-release/release-build.sh

```diff
@@ -81,7 +81,7 @@ BASE_DIR=$(pwd)
 
 MVN="build/mvn --force"
 PUBLISH_PROFILES="-Pmesos -Pyarn -Phive -Phive-thriftserver -Phadoop-2.2"
-PUBLISH_PROFILES="$PUBLISH_PROFILES -Pspark-ganglia-lgpl -Pkinesis-asl"
+PUBLISH_PROFILES="$PUBLISH_PROFILES -Pspark-ganglia-lgpl"
```

Comment:
For this moment, I think you still need this profile, because it causes the non-assembly module to be built and published.